### PR TITLE
New information about disk monitoring

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-662.surge.sh

File: modules/ROOT/pages/monitoring/metrics/expose.adoc [hunk 1]
```diff
@@ -43,6 +43,21 @@ server.metrics.csv.rotation.compression=ZIP
 xref:reference/configuration-settings.adoc#config_server.metrics.csv.rotation.compression[`server.metrics.csv.rotation.compression`] selects the compression scheme to use on the files after rotation.
 Since CSV files are highly compressible, it is recommended to enable compression of the files to save disk space.
 
+== Disks
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/expose.adoc:46]: This should be incorporated into the Essential Metrics page here - https://neo4j-docs-operations-662.surge.sh/operations-manual/5/monitoring/metrics/essential/#_server_load_metrics, not to Expose, as in the Expose metrics, we only list the different options on how the metrics can be exposed.

**Review by renetapopova — CHANGES_REQUESTED** [review]
Thanks, @lidiazuin, for looking into this. I left a comment.

File: modules/ROOT/pages/monitoring/metrics/expose.adoc [hunk 2]
```diff
@@ -7,10 +7,10 @@
 Neo4j supports the following ways of exposing data for monitoring purposes:
 
 * Neo4j Ops Manager -- a UI-based tool that enables a DBA (or any administrator) to monitor, administer, and operate all of the Neo4j DBMSs in an Enterprise.
-* CSV files -- retrieve metrics from CSV files (`true` by default).
-* JMX MBeans -- expose metrics over JMX MBeans (`true` by default).
-* Graphite -- send metrics to https://graphiteapp.org/[Graphite] or any monitoring tool based on the Graphite protocol (`false` by default).
-* Prometheus -- publish metrics for polling as https://prometheus.io/[Prometheus] endpoint (`false` by default).
+* CSV files -- retrieve metrics from CSV files. Enabled by default.
+* JMX MBeans -- expose metrics over JMX MBeans. Enabled by default.
+* Graphite -- send metrics to https://graphiteapp.org/[Graphite] or any monitoring tool based on the Graphite protocol (`disabled` by default).
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/expose.adoc:12]:
```suggestion
* Graphite -- send metrics to https://graphiteapp.org/[Graphite] or any monitoring tool based on the Graphite protocol. Disabled by default.
```

File: modules/ROOT/pages/monitoring/metrics/essential.adoc [hunk 3]
```diff
@@ -39,9 +39,11 @@ For more information, see xref:performance/memory-configuration.adoc[].
 | Observe the rate of your data growth so you can plan for additional storage before you run out.
 This applies to all disks that Neo4j is writing to.
 You might also choose to write the log files to a different disk.
+[NOTE]
+An out of disk event may disrupt system availability and cause the database going offline, thus creating the risk of database or log file corruption. To avoid that, system monitoring tools should be configured to monitor available disk space on all drives used for databases, indexes, and transactions logs.
+See more recommendations about xref:performance/disks-ram-and-other-tips.adoc#performance-storage[system storage practices] and xref:clustering/monitoring/show-servers-monitoring.adoc[monitoring servers] and xref:clustering/monitoring/show-databases-monitoring.adoc[databases] in a clustering state.
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/essential.adoc:44]:
```suggestion
For more recommendations, see xref:performance/disks-ram-and-other-tips.adoc#performance-storage[Disks, RAM and other tips], as well as xref:clustering/monitoring/show-servers-monitoring.adoc[Monitoring servers] and xref:clustering/monitoring/show-databases-monitoring.adoc[Monitoring databases] in a Neo4j cluster.
```

**Review by renetapopova — CHANGES_REQUESTED** [review]
Thanks, @lidiazuin! I added some comments, and you also need to add the link to https://neo4j.com/docs/operations-manual/current/clustering/monitoring/endpoints/[Monitor cluster endpoints for status information] at line 113.

**Review by renetapopova — APPROVED** [review]
Approved ✅

**Pull request merged**